lib/foreman_maintain/concerns/system_helpers.rb

```diff
@@ -217,6 +208,60 @@ def el8?
       def el_major_version
         return os_facts['os']['release']['major'] if el?
       end
+
+      def foreman_plugin_prefix
+        if el7?
+          'tfm-rubygem-foreman_'
+        elsif el8?
+          'rubygem-foreman_'
+        elsif debian?
+          'ruby-foreman-'
+        end
+      end
+
+      def smart_proxy_plugin_prefix
+        if el7?
```

Comment:
the proxy was SCL'ed in Foreman 2.0, before that, the packages were also `rubygem-smart_proxy…`

Reply:
Of course, if we don't need to support pre-2.0 versions, that's nice ;)